oceanbench/core/lagrangian_trajectory.py

```diff
@@ -191,7 +192,108 @@ def _build_field_set(dataset) -> FieldSet:
     )
 
 
-def _get_all_particles_positions(
+
+def _get_all_particles_positions(dataset: xarray.Dataset, lats: numpy.ndarray, lons: numpy.ndarray) -> xarray.Dataset:
```

Comment:
```suggestion
def _get_all_particles_positions(dataset: xarray.Dataset, latitudes: numpy.ndarray, longitudes: numpy.ndarray) -> xarray.Dataset:
```

Complete variable names everywhere